main.tf

```diff
@@ -369,8 +369,7 @@ resource "aws_vpc_security_group_egress_rule" "streamlit_alb_sg_alb_all_traffic"
 ################################################################################
 # Create ALB
 resource "aws_lb" "streamlit_alb" {
-  count = 1 
-
+  count = 1
```

Comment:
last small thing - this line should be removed. the module should create the ALB regardless if the user uses the default vpc resources, or references their own. When removing this line, also make sure than any references to `aws_lb.streamlit_alb[0]` is updated to `aws_lb.streamlit_alb` (without the 0 index)